R/which_element.R

```diff
@@ -1,7 +1,7 @@
 #' which_element
 #'
 #' This function returns locations within an EML list object that contain an instance where \code{element == value}
-#' @param eml an EML list object
+#' @param eml_list an EML list object
 #' @param element element to evaluate
```

Comment:
Add class types to parameter definitions.  For instance `#' @param eml_list (S4) an EML list object`